packages/ocs/dashboards/persistent-external/fallback-hook.ts

```diff
@@ -0,0 +1,33 @@
+import { OCSStorageClusterModel } from '@odf/shared/models';
+import { StorageClassResourceKind } from '@odf/shared/types';
+import { referenceForModel } from '@odf/shared/utils';
+import {
+  WatchK8sResource,
+  useK8sWatchResource,
+} from '@openshift-console/dynamic-plugin-sdk';
+
+export const storageClusterResource: WatchK8sResource = {
+  isList: true,
+  kind: referenceForModel(OCSStorageClusterModel),
+};
+
+const fsProvisionerName = '.cephfs.csi.ceph.com';
+
+const useFallbackHook = (storageClusterName) => {
+  const [storageClasses, storageClassesLoaded, storageClassLoadError] =
+    useK8sWatchResource<StorageClassResourceKind[]>(storageClusterResource);
```

Comment:
don't u wanna pass StorageClass resource instead...
```suggestion
    useK8sWatchResource<StorageClassResourceKind[]>(storageClassResource);
```

Reply:
yes. mistake